src/Sections/SectionYourGovernment.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {
+  IconHeading,
+  EmphasizedImage
+} from "@baltimorecounty/dotgov-components";
+const SectionYourGovernment = props => (
```

Comment:
Think naming this `YourGovernmentSection` reads a bit easier.

Reply:
Yeah I think that makes it more consistent with the actual section

Reply:
adjusted on behalf of @tmccoy529